package.json

```
@@ -47,6 +47,7 @@
     "dependencies": {
         "@erc725/erc725.js": "^0.20.0",
         "@lukso/lsp-smart-contracts": "^0.11.1",
+        "@openzeppelin/contracts": "^5.0.1",
```

I suggest we keep the same version as the one used in `@lukso/lsp-smart-contracts`, which is 4.9.2

```suggestion
        "@openzeppelin/contracts": "^4.9.2",
```